util/README.md

```diff
@@ -1,5 +1,13 @@
 # Utility scripts
 
+## Check tool directory names
+
+Verify that the path of all tools contains the name of the corresponding tool shed repository:
```

Comment:
This will not be possible for suites. In suites toolshed repos are made up by the tool id.

Reply:
Can you please explain this a little bit more or point me where to read about it? I have an intuitive understanding of what a "suite" is, but it is not listed as a term on https://galaxyproject.org/tools and in the this context here, I'm not sure what you mean.

Reply:
Internet is really bad here... A suite is for example Fiji. Many tools in one folder that are decomposed by planemo into separate toolshed repos. One GitHub folder, shared content, eg macros, but separate tools. And usually the toolshed name is made up automatically by the tool ID

Reply:
Thanks for the clarifications! I understand that there are cases, where the policy that _"the path of a tool should contain the name of the tool shed repo"_ doesn't make sense. But I think it is a good practice in order to increase the findability of the tool sources for those tools, where it _is_ possible.

In fact, there currently is one tool, for which the above mentioned policy is not fulfilled, because of an unfortunate typo: `tools/voronoi_tessellation/.shed.yml` (correct spelling) has `name: voronoi_tesselation` (wrong spelling). As I understand that it's not possible to rename the tool shed repo, we cannot fix this, right?